content/en/database_monitoring/guide/aurora_autodiscovery.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Automatically configure your aurora clusters for monitoring
+kind: guide
+---
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Aurora cluster auto discovery is in **beta**. If you have any feedback about this feature, please [contact support](https://docs.datadoghq.com/help/).
+{{< /beta-callout >}}
+
+This guide assumes that you have configured [Database Monitoring][1] for your Aurora hosts.
+
+## Before you begin
+
+Supported databases
+: Postgres, MySQL
+
+Supported Agent versions
+: 7.53.0+ (Not currently released)
+
+## Agent Installation
+
+// TODO: Add the installation steps for the beta agent
+
+## Overview
+
+The Datadog Agent supports configuring automatic discovery and monitoring of your aurora clusters. This feature works in lieu of manually configuring the Agent by listing individual database host endpoints.
+
+This is helpful for environments where the number of endpoints can be dynamic, such as with clusters that use [Aurora Auto Scaling][6].
```

Comment:
Auto Scale will be the only reason that causes dynamic number of hosts right? Wonder if we should just say "This is helpful for Aurora clusters with auto-scaling enabled, automatically adjusting the number of read replicas in response to workload variations."